src/streaming-client.ts

```diff
@@ -53,14 +57,20 @@ export class RevAiStreamingClient extends EventEmitter {
      * Begins a streaming connection. Returns a duplex
      * from which the user can read responses from the api and to which the user
      * should write their audio data
+     * @param config (Optional) Optional configuration for the streaming session
      *
      * @returns BufferedDuplex. Data written to this buffer will be sent to the api
      * Data received from the api can be read from this duplex
      */
-    public start(): Duplex {
+    public start(config?: SessionConfig): Duplex {
         let url = this.baseUrl +
             `?access_token=${this.accessToken}` +
-            `&content_type=${this.config.getContentTypeString()}`;
+            `&content_type=${this.config.getContentTypeString()}` +
+            `&user_agent=${encodeURIComponent(`RevAi-NodeSDK/${sdkVersion}`)}`;
```

Comment:
Please double check on the server side that the uri is properly decoded.

Reply:
Should ``RevAi-NodeSDK/${sdkVersion}`` be defined as a constant somewhere?

Reply:
We don't really have a global constants file in the SDK and I don't think it's necessary for now. I don't forsee us having more than 2 uses of this constant string anytime in the near future